app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,128 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      A modern languages teacheer opens young minds to the languages and cultures around the world. </p>
+      <p>As we strive to achieve our vision for a 'Global Britain' mutual inter-cultural understanding and insights through the fostering of a shared language, is vital.</p>
+      <p>Qualifications are available in 18 modern languages with the must popular being French, German and Spanish in key stage 2 and 3.</p>
+  <p>Tax-free scholarships of £27,000 and bursaries of £25,000 are available for eligible trainee language teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "",
+      colour: "white",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+  
+    <% end %>
+  </section>
+
+    <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "As a language teacher, you can light the flame of language learning and build cultural capital for all young people",
```

Comment:
@caitlin-reidy I don't really understand what the 2nd half of this quote means, **build cultural capital for all young people**?